src/lib/Scenes/Artwork/Components/ArtworkActions.tsx

```diff
@@ -130,17 +130,16 @@ export class ArtworkActions extends React.Component<ArtworkActionsProps> {
             </TouchableWithoutFeedback>
           )}
 
-          {Constants.AREnabled &&
-            is_hangable && (
-              <TouchableWithoutFeedback onPress={() => this.openViewInRoom()}>
-                <UtilButton pr={3}>
-                  <Box mr={0.5}>
-                    <EyeOpenedIcon />
-                  </Box>
-                  <Sans size="3">View in Room</Sans>
-                </UtilButton>
-              </TouchableWithoutFeedback>
-            )}
+          {Constants.AREnabled && is_hangable && (
```

Comment:
I think there's no change in here, right? (No objection to the formatting change, just verifying that I'm not missing anything.)

Reply:
I think its just line spacing changes from the linter.